src/components/TopBar.js

```diff
@@ -1,9 +1,8 @@
-import { Fragment } from 'react'
+import { Collapse } from 'react-collapse'
```

Comment:
We also have a src/components/Collapse, which could get confusing.

Reply:
Ooh, I forgot about that. Let me combine these.